envision/web/src/components/simulation.js

```diff
@@ -185,10 +185,11 @@ export default function Simulation({
           // React to trigger update, 0.1 millisecond
           await sleep(0.1);
         } else if (playingMode == PLAYMODES.near_real_time) {
-          await sleep(
-            msInSec * (elapsed_times[0] - prevElapsedTime) -
-              (Date.now() - waitStartTime)
-          );
+          let delt_elapsed_time_in_sec = msInSec * (elapsed_times[0] - prevElapsedTime);
+          if (delt_elapsed_time_in_sec > 100) {
+            delt_elapsed_time_in_sec = 100
```

Comment:
After using Python, it's easy to forget about semicolons. Though not always necessary in js, they should be used to make sure statements stay separate from each other.
```suggestion
            delt_elapsed_time_in_sec = 100;
```